observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,53 @@
+After doing so, users can then monitor their own services and workloads without the need for an additional monitoring solution.
+
+Depending on which users will use the monitoring stack and for what purposes, as a cluster administrator, you can configure monitoring components to suit the needs of different users in various scenarios.
+
+[id="core-platform-monitoring-first-steps_{context}"]
+== Core platform monitoring: First steps
+
+After {product-title} is installed, cluster administrators typically perform certain basic activities to configure core platform monitoring to suit their needs.
+These activities include configuring options for Prometheus and Alertmanager, setting up short-term and long-term storage, and adding service monitors.
+When you have completed these configuration activities, users can query and view collected metrics and receive alerts.
+
+* xref:../../observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects[Assign monitoring cluster roles] to any non-administrator users that need to access certain monitoring features.
+The `monitoring-edit` cluster role permits users to create alerts and to create, modify, and delete `ServiceMonitor` and `PodMonitor` resources to scrape metrics from services and pods.
+The `alert-routing-edit` cluster role permits users to route alerts.
+* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#creating-cluster-monitoring-configmap_configuring-the-monitoring-stack[Create the `cluster-monitoring-config` `ConfigMap` object] if it does not exist.
+* If needed, xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#assigning-tolerations-to-monitoring-components_configuring-the-monitoring-stack[assign tolerations] to monitoring stack components to enable moving them to tainted nodes.
+* If needed, xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#setting-the-body-size-limit-for-metrics-scraping_configuring-the-monitoring-stack[set the body size limit] for metrics collection to help avoid situations in which Prometheus consumes excessive amounts of memory when scraped targets return a response that contains a large amount of data.
+* For short-term data retention, xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#configuring-persistent-storage_configuring-the-monitoring-stack[configure persistent storage] by creating Persistent Volume Claims (PVCs) to provision persistent storage volumes (PVs).
+Configure these PVs to be used by Prometheus to store metrics data and by Alertmanager to store alert data. Specify the metrics data retention parameters for Prometheus and Thanos Ruler.
++
+[NOTE]
+====
+By default, a newly installed {product-title} system is configured to fire an alert named `PodStartupStorageOperationsFailing` to remind you that storage is not configured.
```

Comment:
did not find `PodStartupStorageOperationsFailing` alert
```
$ oc get pv
No resources found

$ token=`oc create token prometheus-k8s -n openshift-monitoring`
$ oc -n openshift-monitoring exec -c prometheus prometheus-k8s-0 -- curl -k -H "Authorization: Bearer $token" 'https://thanos-querier.openshift-monitoring.svc:9091/api/v1/query?' --data-urlencode 'query=ALERTS{alertname="PodStartupStorageOperationsFailing"}' | jq
{
  "status": "success",
  "data": {
    "resultType": "vector",
    "result": [],
    "analysis": {}
  }
}
```
maybe you meant if we don't configure PV for prometheus, there's PrometheusDataPersistenceNotConfigured message in clusteroperator monitoring?
```
$ oc get clusteroperator monitoring -oyaml
...
  - lastTransitionTime: "2024-07-17T00:18:44Z"
    message: 'Prometheus is running without persistent storage which can lead to data
      loss during upgrades and cluster disruptions. Please refer to the official documentation
      to see how to configure storage for Prometheus: https://docs.openshift.com/container-platform/4.16/monitoring/configuring-the-monitoring-stack.html'
    reason: PrometheusDataPersistenceNotConfigured
    status: "False"
    type: Degraded
```

Reply:
Changed to:

`By default, in a newly installed {product-title} system, the `ClusterOperator` resource reports a `PrometheusDataPersistenceNotConfigured` status message to remind you that storage is not configured.`